spec/fixtures/template_inject/nginx_kong_test_tcp_echo_server_custom_inject_stream.lua

```diff
@@ -0,0 +1,23 @@
+return [[
+server {
+    listen 8188;
+    listen 8189 ssl;
+
+> for i = 1, #ssl_cert do
+    ssl_certificate     $(ssl_cert[i]);
+    ssl_certificate_key $(ssl_cert_key[i]);
+> end
+    ssl_protocols TLSv1.1 TLSv1.2 TLSv1.3;
+
+    content_by_lua_block {
+        local sock = assert(ngx.req.socket())
+        local data = sock:receive()  -- read a line from downstream
+        if data then
+            sock:send(data.."\n") -- echo whatever was sent
+            ngx.log(ngx.INFO, "received data: " .. data)
+        else
+            ngx.log(ngx.WARN, "Nothing received")
+        end
+    }
```

Comment:
Is there a reason why we cannot just use this:
https://github.com/Kong/kong/blob/master/spec/fixtures/template_inject/nginx_kong_test_custom_inject_stream.lua#L13-L29

Or modify that to support also the reports tests?

It looks like both are kinda echo servers with minor differences.

Reply:
I was thinking for making the scope small, this PR tries to not touch existing functionality if possible. 

And another reason is there are another 47 tcp_server() usage needs to be removed in our testing. And there may be other required modification on this `nginx_kong_test_tcp_echo_server_custom_inject_stream.lua` file. To enable the later on change be separated from existing one, this PR puts the change in the new file.

Reply:
As discussed offline, this is good, and I agree with your points here.